{{cookiecutter.code}}/{{cookiecutter.project_slug}}/static/{{cookiecutter.project_slug}}/css/customize.css

```diff
@@ -0,0 +1,8 @@
+.juntagrico-logo {
+    /* uncomment the next line and add your logo in the img folder */
+    /* background: url(/static/{{cookiecutter.project_slug}}/img/logo.png) no-repeat 0 0; */
+    background-size: contain;
```

Comment:
have you tested it that this work and not no logo is shown?
it shoudl work but better safe than sorry

Reply:
Yes. Tested by commenting out that line on my own instance locally.